### DISCUSSION TITLE
Release date for next version of excalidraw

### DISCUSSION THREAD

**Discussion by pvaliyaveedan** [excalidraw/excalidraw#3856] [Q&A] [answered]
Do you guys have a release date for the next version of npm package with the new undo/redo button changes?

**Comment by ad1992** [comment, marked as answer]
@pvaliyaveedan official release would take some time but you can use [@excalidraw/excalidraw-next](https://www.npmjs.com/package/@excalidraw/excalidraw-next) if you need the latest changes. You might want to pin the version since we release this package with every change :)

**Reply by pvaliyaveedan** [reply]
Awesome! Thank you!
